metabase/metabase
# fix(sdk): Upgrade outdated dependencies to remove installation warnings

Closes part of https://github.com/metabase/metabase/issues/44772

### Description

We want to fix warnings shown during dependencies installation using SDK npm package.

**Before:**
```
warning "@metabase/embedding-sdk-react > @visx/axis@1.8.0" has incorrect peer dependency "react@^16.3.0-0".
warning "@metabase/embedding-sdk-react > @visx/grid@1.16.0" has incorrect peer dependency "react@^15.0.0-0 || ^16.0.0-0".
warning "@metabase/embedding-sdk-react > @visx/group@1.7.0" has incorrect peer dependency "react@^15.0.0-0 || ^16.0.0-0".
warning "@metabase/embedding-sdk-react > @visx/text@1.7.0" has incorrect peer dependency "react@^16.3.0-0".
warning "@metabase/embedding-sdk-react > @visx/axis > @visx/shape@1.8.0" has incorrect peer dependency "react@^16.3.0-0".
warning "@metabase/embedding-sdk-react > @visx/grid > @visx/shape@1.16.0" has incorrect peer dependency "react@^16.3.0-0".
warning "@metabase/embedding-sdk-react > postcss-discard-comments@6.0.2" has unmet peer dependency "postcss@^8.4.31".
warning "@metabase/embedding-sdk-react > react-motion@0.5.2" has incorrect peer dependency "react@^0.14.9 || ^15.3.0 || ^16.0.0".
warning "@metabase/embedding-sdk-react > react-textarea-autosize@5.2.1" has incorrect peer dependency "react@>=0.14.0 <17.0.0".
warning "@metabase/embedding-sdk-react > react-router@3.2.6" has incorrect peer dependency "react@^0.14.0 || ^15.0.0 || ^16.0.0".
warning "@metabase/embedding-sdk-react > kbar > react-virtual@2.10.4" has incorrect peer dependency "react@^16.6.3 || ^17.0.0".

```


**After:**
```
warning "@metabase/embedding-sdk-react > react-router@3.2.6" has incorrect peer dependency "react@^0.14.0 || ^15.0.0 || ^16.0.0".
warning "@metabase/embedding-sdk-react > kbar > react-virtual@2.10.4" has incorrect peer dependency "react@^16.6.3 || ^17.0.0".
```

### How to verify

Try to install shoppy app with package built from this commit:

- It should have fewer warnings.
- CI should pass.
- No visual regressions should happen.
